backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/manufacturing/job/service/ManufacturingJobLoaderAndSaverSupportingServices.java

```diff
@@ -106,9 +107,15 @@ public Optional<HuId> getHuIdByQRCodeIfExists(@NonNull final HUQRCode qrCode)
 		return huQRCodeService.getHuIdByQRCodeIfExists(qrCode);
 	}
 
-	public void assignQRCode(@NonNull HUQRCode qrCode, @NonNull HuId huId)
+	public void assignQRCodeForReceiptHU(@NonNull final HUQRCode qrCode, @NonNull final HuId huId)
```

Comment:
why mutating (i.e. assigning an QR code) is part of a load/save support service?
imho this is not the right place for this method

Reply:
I don't know, I've just adapted the code to use the huassignment, the assign was already here, I could move it ofc, but I don't think it's wihtin scope